### ISSUE TITLE
Unspecified error [353ba28057]

### ISSUE THREAD

**Issue by aem-intellij-plugin-error-reporter** [aem-intellij-plugin-error-reporter/auto-generated-issues-storage#76]
## Description
*No description*
## Exception
```
java.lang.IllegalStateException: project.baseDir must not be null
	at co.nums.intellij.aem.fj.<init>(Unknown Source)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
	at com.intellij.serviceContainer.ConstructorInjectionKt.instantiateUsingPicoContainer(constructorInjection.kt:53)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClassWithConstructorInjection(ComponentManagerImpl.kt:660)
	at com.intellij.serviceContainer.ServiceComponentAdapter.createAndInitialize(ServiceComponentAdapter.kt:54)
	at com.intellij.serviceContainer.ServiceComponentAdapter.doCreateInstance(ServiceComponentAdapter.kt:41)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstanceUncached(BaseComponentAdapter.kt:115)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstance(BaseComponentAdapter.kt:69)
	at com.intellij.serviceContainer.ComponentManagerImpl.doGetService(ComponentManagerImpl.kt:411)
	at com.intellij.serviceContainer.ComponentManagerImpl.getService(ComponentManagerImpl.kt:394)
	at com.intellij.openapi.components.ServiceManager.getService(ServiceManager.java:24)
	at co.nums.intellij.aem.fo.a(Unknown Source)
	at co.nums.intellij.aem.E.a(Unknown Source)
	at co.nums.intellij.aem.bo.getLanguage(Unknown Source)
	at com.intellij.psi.LanguageSubstitutors.substituteLanguage(LanguageSubstitutors.java:56)
	at com.intellij.lang.LanguageUtil.getLanguageForPsi(LanguageUtil.java:38)
	at com.intellij.psi.impl.file.impl.FileManagerImpl.createFileViewProvider(FileManagerImpl.java:250)
	at com.intellij.psi.impl.file.impl.FileManagerImpl.findViewProvider(FileManagerImpl.java:196)
	at com.intellij.psi.impl.file.impl.FileManagerImpl.findFile(FileManagerImpl.java:359)
	at com.intellij.psi.impl.PsiDocumentManagerBase.getPsiFile(PsiDocumentManagerBase.java:145)
	at com.intellij.psi.impl.PsiDocumentManagerBase.getPsiFile(PsiDocumentManagerBase.java:92)
	at com.intellij.psi.impl.PsiDocumentManagerImpl.getPsiFile(PsiDocumentManagerImpl.java:64)
	at com.intellij.openapi.editor.impl.SettingsImpl.getPsiFile(SettingsImpl.java:408)
	at com.intellij.openapi.editor.impl.SettingsImpl.getTabSize(SettingsImpl.java:380)
	at com.intellij.openapi.editor.ex.util.EditorUtil.getTabSize(EditorUtil.java:328)
	at com.intellij.openapi.editor.impl.view.EditorView.getTabSize(EditorView.java:553)
	at com.intellij.openapi.editor.impl.view.LogicalPositionCache.reset(LogicalPositionCache.java:75)
	at com.intellij.openapi.editor.impl.view.EditorView.reinitSettings(EditorView.java:350)
	at com.intellij.openapi.editor.impl.EditorImpl.<init>(EditorImpl.java:496)
	at com.intellij.openapi.editor.impl.EditorFactoryImpl.createEditor(EditorFactoryImpl.java:201)
	at com.intellij.openapi.editor.impl.EditorFactoryImpl.createEditor(EditorFactoryImpl.java:161)
	at com.intellij.openapi.fileEditor.impl.text.TextEditorComponent.createEditor(TextEditorComponent.java:133)
	at com.intellij.openapi.fileEditor.impl.text.TextEditorComponent.<init>(TextEditorComponent.java:79)
	at com.intellij.openapi.fileEditor.impl.text.PsiAwareTextEditorImpl$PsiAwareTextEditorComponent.<init>(PsiAwareTextEditorImpl.java:123)
	at com.intellij.openapi.fileEditor.impl.text.PsiAwareTextEditorImpl$PsiAwareTextEditorComponent.<init>(PsiAwareTextEditorImpl.java:116)
	at com.intellij.openapi.fileEditor.impl.text.PsiAwareTextEditorImpl.createEditorComponent(PsiAwareTextEditorImpl.java:101)
	at com.intellij.openapi.fileEditor.impl.text.TextEditorImpl.<init>(TextEditorImpl.java:50)
	at com.intellij.openapi.fileEditor.impl.text.PsiAwareTextEditorImpl.<init>(PsiAwareTextEditorImpl.java:42)
	at com.intellij.openapi.fileEditor.impl.text.PsiAwareTextEditorProvider.createEditor(PsiAwareTextEditorProvider.java:35)
	at com.intellij.ide.lightEdit.LightEditorManagerImpl.createFileEditor(LightEditorManagerImpl.java:69)
	at com.intellij.ide.lightEdit.LightEditorManagerImpl.doCreateEditor(LightEditorManagerImpl.java:52)
	at com.intellij.ide.lightEdit.LightEditorManagerImpl.createEditor(LightEditorManagerImpl.java:91)
	at com.intellij.ide.lightEdit.LightEditServiceImpl.doOpenFile(LightEditServiceImpl.java:145)
	at com.intellij.ide.lightEdit.LightEditServiceImpl.lambda$openFile$1(LightEditServiceImpl.java:112)
	at com.intellij.ide.lightEdit.LightEditServiceImpl.invokeOnEdt(LightEditServiceImpl.java:134)
	at com.intellij.ide.lightEdit.LightEditServiceImpl.doWhenActionManagerInitialized(LightEditServiceImpl.java:128)
	at com.intellij.ide.lightEdit.LightEditServiceImpl.openFile(LightEditServiceImpl.java:111)
	at com.intellij.ide.lightEdit.LightEdit.openFile(LightEdit.java:25)
	at com.intellij.ide.CommandLineProcessor.doOpenFile(CommandLineProcessor.java:77)
	at com.intellij.ide.CommandLineProcessor.doOpenFileOrProject(CommandLineProcessor.java:46)
	at com.intellij.ide.CommandLineProcessor.processExternalCommandLine(CommandLineProcessor.java:202)
	at com.intellij.idea.ApplicationLoader$addActivateAndWindowsCliListeners$2$1.run(ApplicationLoader.kt:333)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:201)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:831)
	at com.intellij.openapi.application.impl.ApplicationImpl.lambda$invokeAndWait$8(ApplicationImpl.java:480)
	at com.intellij.openapi.application.impl.LaterInvocator$1.run(LaterInvocator.java:124)
	at com.intellij.openapi.application.impl.FlushQueue.doRun(FlushQueue.java:80)
	at com.intellij.openapi.application.impl.FlushQueue.runNextEvent(FlushQueue.java:128)
	at com.intellij.openapi.application.impl.FlushQueue.flushNow(FlushQueue.java:46)
	at com.intellij.openapi.application.impl.FlushQueue$FlushNow.run(FlushQueue.java:184)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:974)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:847)
	at com.intellij.ide.IdeEventQueue.lambda$null$8(IdeEventQueue.java:449)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:741)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$9(IdeEventQueue.java:448)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:831)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:496)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```
## Error context
```
Last Action:       null
Plugin Version:    0.11.4
OS Name:           Windows 10
Java Version:      11.0.6
Java VM Vendor:    JetBrains s.r.o
Product Name:      WebStorm
Full Product Name: WebStorm
App Version Name:  WebStorm
Is EAP:            false
App Build:         WS-201.7223.93
App Version:       2020.1.1
```

**Comment by aem-intellij-plugin-error-reporter** [comment]
## Description
*No description*
## Error context
```
Last Action:       
Plugin Version:    0.8.5
OS Name:           Windows 8
Java Version:      1.8.0_51
Java VM Vendor:    Oracle Corporation
Product Name:      PyCharm
Full Product Name: PyCharm
App Version Name:  PyCharm
Is EAP:            false
App Build:         PY-143.1559
App Version:       5.0.3
```

**Comment by karollewandowski** [comment]
To the last error reporter: please update the plugin.
0.8.5 has broken error reporter and you probably got different stack trace that is reported to any existing issue in this repo.
You can also create issue manually, but it's better to update the plugin. There were a lot of bug fixes released since 0.8.5
